### DISCUSSION TITLE
Please add Linux support - Or MAUI will finally shut down by Microsoft

### DISCUSSION THREAD

**Discussion by xky0007** [dotnet/maui#22990] [General]
If multi-platforms don't include Linux. It's not real multi-platforms. 
Why people would choose MAUI if the app required to publish at Windows, MacOS and Linux? Nobody....

Microsoft will finally shut down MAUI just like SilverLight, UWP and WSA... The only hope is add Linux support.

**Comment by mattleibow** [comment]
Duplicate of https://github.com/dotnet/maui/discussions/339
